sa/database.go

```diff
@@ -45,7 +45,7 @@ type DbSettings struct {
 	ConnMaxIdleTime time.Duration
 }
 
-// InitWrappedDb constructs a wrapped gorp mapping object with the provided
+// InitWrappedDb constructs a wrapped borp mapping object with the provided
```

Comment:
Realizing it's slightly weird that this function is in //sa/database.go, rather than somewhere in //db. None of this function's inputs or outputs seem SA-specific. If it were over there, it wouldn't need the SQLDb() getter method, since it could access the underlying *sql.DB directly with same-package privilege.

Same goes for SetSqlDebug.

Probably best to file a bug to track this for a follow-up, so this PR can remain minimal and mechanical.

Reply:
Yeah good point. I meant to file something about this. https://github.com/letsencrypt/boulder/issues/6991